src/main/java/calculator/parser/InputParser.java

```diff
@@ -0,0 +1,60 @@
+package calculator.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputParser {
+    private final String DEFAULT_DELIMITER_FILTER = "[,:]";
```

Comment:
상수에 static 키워드를 붙여서 클래스끼리 공유하면 좋을 것 같아요!
```suggestion
    private final String DEFAULT_DELIMITER_FILTER = "[,:]";
```

Reply:
static 을 사용하게 되면
프로그램이 실행될 때 프로그램 전역적으로 생성되어
모든 객체가 이를 알 수 있게 된다고 알고 있습니다

이는 객체지향적이지 못하다는 생각이 들었고
그렇기에 상수라고 꼭 static 을 써야 하나?
라는 생각이 들었고

상수라고 해도 클래스끼리 공유하지 않는 경우가 많았기에
꼭 static 을 사용하지 않아도 되겠다는 생각이 들었습니다

그런데 다른 분 의견을 들어보니 private static 으로 만들면
캡슐화가 될 테니 괜찮지 않겠냐는 이야기를 해주셨습니다

이 부분에서 객체지향에 문제가 발생하지는 않는다고 판단해
이번 주 미션부터는 static 을 사용하고자 하는데
그 외에도 상수에는 꼭 static 을 붙여야 하는 이유가 있다고 생각하시는지 여쭤보고 싶습니다!